modules/openapi-generator/src/main/resources/JavaSpring/model.mustache

```diff
@@ -46,7 +46,7 @@ import javax.annotation.Generated;
 {{>enumOuterClass}}
 {{/isEnum}}
 {{^isEnum}}
-{{>pojo}}
+{{#vendorExtensions.x-is-one-of-interface}}{{>oneof_interface}}{{/vendorExtensions.x-is-one-of-interface}}{{^vendorExtensions.x-is-one-of-interface}}{{>pojo}}{{/vendorExtensions.x-is-one-of-interface}}
```

Comment:
No samples use this option currently, so it's hard to evaluate the output currently

Reply:
Good point - will add a sample project